kebechet/config.py

```diff
@@ -222,37 +226,37 @@ def post(*args, **kwargs):
             kwargs.pop("verify", None)
             return original_post(*args, **kwargs, verify=verify)
 
-        requests.Session.post = post
+        requests.Session.post = post  # type: ignore
```

Comment:
Maybe we could drop these as we dropped igitt (see the comment above).

Reply:
Can we remove `_tls_verification` method in config.py now that we don't use IGitt?

Reply:
Yes, I think so :+1: